src/view/view.js

```diff
@@ -361,12 +361,14 @@ export default class View {
 		callback( this._writer );
 		this._ongoingChange = false;
 
-		// Execute all document post-fixers after the change.
-		this._postFixersInProgress = true;
-		this.document._callPostFixers( this._writer );
-		this._postFixersInProgress = false;
-
+		// This lock is used by editing controller to render changes from outer most modelchange() once. As plugins might call
+		// view.change() inside model.change() block - this will ensures that postfixers are called once before rendering.
```

Comment:
`ensures that postfixers are called once before rendering.` -> `ensures that postfixers and rendering are called once after all changes.`